### PR TITLE
Ports over the package select value props component

### PR DESCRIPTION
**Pull request by roblevintennis**
- [ ] Allow for passing in custom iconUrl, header, and subHeader and set the current ones to defaults

**Description:**
- [Asana Task](https://app.asana.com/0/1136962714401929/1150572630928081/f)
- Attempting to port over the value props component to EDS

**Referencing PR or Branch (e.g. in CMS or monorepo):**
- https://github.com/getethos/ethos/pull/1557

**Screenshots:**

Not sure what it's supposed to look like but:

image

image

### CODE REVIEW

**Comment by roblevintennis** [comment]
@jameshawkinsjr so looking again at the code:

```js

// iconUrl, header, and subHeader

const sections = [
  {
    iconUrl:
      'https://res.cloudinary.com/getethos/image/upload/v1566238234/02_Icons/icon_hourglass_xedxvp.svg',
    header: 'Fast, easy, free application',
    subHeader:
      'Our online application is straightforward and can be completed in minutes.',
    alt: 'Hourglas Icon',
  },
  {
    iconUrl:
      'https://res.cloudinary.com/getethos/image/upload/v1566238232/02_Icons/icon_half-dollar_wsrssk.svg',
    header: 'No cancellation fees',
    subHeader:
      "You can cancel your policy anytime, with no extra fees. If you change your mind within the first 30 days, we'll refund your payment in full.",
  },
  {
    iconUrl:
      'https://res.cloudinary.com/getethos/image/upload/v1566238234/02_Icons/icon_heart-shield_sh7goi.svg',
    header: 'Trusted Partners',
    subHeader:
      'We partner with Legal & General America, RGAX, Assurity Life Insurance Company, and Munich Re to provide you affordable coverage.',
    alt: 'Hear Shield Icon',
  },
]
```

so I think I'll just make sections properties and use these as default

**Comment by roblevintennis** [comment]
image

**Comment by roblevintennis** [comment]
@jameshawkinsjr @zach-ethos @tkarssli can I get a final review on this guy?

File: src/components/ValueProps/ValueProps.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    subHeader:
+      "You can cancel your policy anytime, with no extra fees. If you change your mind within the first 30 days, we'll refund your payment in full.",
+  },
+  {
+    iconUrl:
+      'https://res.cloudinary.com/getethos/image/upload/v1566238234/02_Icons/icon_heart-shield_sh7goi.svg',
+    header: 'Trusted Partners',
+    subHeader:
+      'We partner with Legal & General America, RGAX, Assurity Life Insurance Company, and Munich Re to provide you affordable coverage.',
+    alt: 'Hear Shield Icon',
```
Comment (jameshawkinsjr) [inline src/components/ValueProps/ValueProps.js]: `Heart Shield Icon` ?

**Pull request merged**